nahusenayElias/Team_4
# DRD-49: Mautic Contact Form

## Related ticket / Customer approval:
[DRD-49](https://edu-team4-react24k.atlassian.net/browse/DRD-49?atlOrigin=eyJpIjoiZTVjMjI0ODc5NmExNDZmYWI5YWQwNjI0YmFmNDZjNWYiLCJwIjoiaiJ9)
## In this PR:
- Created mauticContactForm inside Mautic
- Render form in an iframe on Contact page
# IMPORTANT - First backup your current mautic db:
- cd to mautic-lando
- `lando db-export mauticBackup241107` or something like that
- move the backup outside the repo somewhere for safekeeping

## Testing instructions:

- Checkout branch
- cd to mautic-lando
- `lando db-import mautic-DRD-49.gz`
- You might have to do lando restart for the mautic instance at this point
- Check Mautic > Forms. You should see mauticContactForm
- Fill in the form on frontend and submit it, check you get the submission on Mautic
- You should also see a new non anonymous contact (click off Toggle anonymous contacts)
(- If none of this work, import the database from your backup)

Notice: currently the form creates a new Contact with a new id instead of binding the form data to an existing contact.

[DRD-49]: https://edu-team4-react24k.atlassian.net/browse/DRD-49?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
